srcpkgs/docker/template

```diff
@@ -54,7 +54,7 @@ do_install() {
 	vbin components/cli/build/docker docker
 	vinstall components/cli/contrib/completion/bash/docker 644 usr/share/bash-completion/completions
 	vinstall components/cli/contrib/completion/zsh/_docker 644 usr/share/zsh/site-functions
-	vinstall components/cli/contrib/completion/fish/docker.fish 644 usr/share/fish_vendor_completions.d/docker.fish
+	vinstall components/cli/contrib/completion/fish/docker.fish 644 usr/share/fish/vendor_completions.d
```

Comment:
either leave as is or change all of similar commands in the same manner.

Reply:
Just the fish completion file is installed to the wrong path:
`/usr/share/fish_vendor_completions.d/docker.fish/docker.fish`
instead of 
`/usr/share/fish/vendor_completions.d/docker.fish`
because of this typo:
```
fish_vendor_completions.d
    ^
```

Reply:
indeed, i didn't notice. still, please add some consistency

Reply:
You mean like using `cd`, similar to the other functions or what else should be more consistent?

Reply:
no. either provide the explicit file name by appending it to all vinstall's second arg or do not. just do it consistently